[IMP] pos: rename & redo receipt page (structure, text, images) (#3429)
odoo/documentation: content/applications/sales/point_of_sale/shop/posreceipts.rst

========
Receipts
========

Configuration
=============

Set up receipts by going to :menuselection:`Point of Sale --> Configuration --> Point of Sale`,
selecting a POS, and scrolling down to the :guilabel:`Bills & Receipts` section.

> To **customize** the **header** and **footer**, activate :guilabel:`Header & Footer`. Doing so
> enables fields to fill in with the information you want to be printed on the receipts. To **print
> receipts** automatically once the payment is registered, enable the :guilabel:`Automatic Receipt
> Printing` setting.

Reviewer: ```suggestion
To **customize** the **header** and **footer**, activate :guilabel:`Header & Footer` and
fill in both fields with the information to be printed on the receipts.
To **print receipts** automatically once the payment is registered, enable the :guilabel:`Automatic Receipt
Printing` setting.
```
I think the first part could be simplified, just a suggestion though.
I would put the second part in a new paragraph, as it's not directly related to the first part.

.. image:: receipts/receipt.png
   :align: center
   :alt: pos receipt

.. seealso::
   - :doc:`../restaurant/bill_printing`
   - :doc:`../restaurant/split`

Reprint a receipt
=================

> From the POS interface, click :guilabel:`Orders`, open the dropdown selection menu next to the

Reviewer 2: ```suggestion
From the POS interface, click :guilabel:`Orders` at the top, open the dropdown selection menu next to the
```

Author: it would suggest there's another "orders" on the POS interface, I don't think it's necessary to precise where the order button is situated

> From the POS interface, click :guilabel:`Orders`, open the dropdown selection menu next to the

Reviewer 2: ```suggestion
From the POS interface, click :guilabel:`Orders`, open the dropdown selection menu next to the
```
Where is this ? Be it the dashboard or the POS interface, I cannot find any dropdown selection menu. On the *dashboard*, I can do :menuselection:`Orders --> ??`, and on the POS *interface*, I can only click :guilabel:`Quotation/Order`

Author: explained on discord :)

> search bar, and change the default :guilabel:`All active orders` to :guilabel:`Paid`. Then, select

Reviewer: ```suggestion
search bar, and change the default :guilabel:`All active orders` filter to :guilabel:`Paid`. Then, select
```
Specifying "filter" adds some clarity I feel
